Dockerfile

```diff
@@ -12,6 +12,9 @@ RUN ln -s /usr/local/kotlinc/bin/kotlin /usr/local/bin
 RUN ln -s /usr/local/kotlinc/bin/kotlinc /usr/local/bin
 RUN kotlin -e 'println("Hello from Kotlin")'
 
+RUN echo > /root/.bashrc
```

Comment:
I think I wrote it this way, but now I wonder if it's not better to either `EXPORT` it or write it to a file and `source ..` it.

@sdpryan What do you think?

Reply:
I don't entirely understand what you mean. These two lines are writing some shell configuration to make it look pretty, right?

What problem are you trying to solve by changing it, and is it worth time changing?

Or do you mean the `run_kts` stuff?

Reply:
I mean it looks a bit ugly, because it's overriding the `.bashrc` file every time, which might cause trouble in the future if something else is as well. While it's fine for python, i'm not sure it will be across the board.

Reply:
I think it's not worth worrying about changing right now, honestly. To prevent someone copy and pasting it in future without thinking about it, add a comment with your concerns.